gdal/PROVENANCE.TXT

```diff
@@ -139,56 +139,21 @@ Note: all the following are build options, not required.
                Derived from modules of GRASS 4.1 (public domain), properly noted in the headers.
                See http://trac.osgeo.org/gdal/ticket/2975
 
-=== gdal/debian ===
-
-* copyright somewhat unclear, not actually part of software.
-
 === gdal/doc ===
 
 * no copyright messages in .dox files.
 * ERMapper logo used with permissions.
 * "ru" subdirectory (Russian translations) by Andrey Kiselev.
 
-=== gdal/pymod ===
-
-* gdal_merge.py: copyright held by Atlantis, under LGPL. This is an acceptable license and it is just noted as an exception to the general MIT/X rule for GDAL but no further action required.
-* gdal2xyz.py: copyright me, under LGPL - relicensed MIT/X.
-* gdal_wrap.c: Generated with SWIG under very permissive license (noted in file).
-* Scripts mostly by me, all ok.
-
-=== gdal/pymod/samples ===
-
-* gdal_vrtmerge.py: Under LGPL (derived from gdal_merge.py), several contributors.
-* histrep.py: corrected copyright message (Atlantis to FrankW), change license to MIT/X.
-* various contributors, all under proper and compatible copyright headers.
-
-=== gdal/ogr/ogrsf_rmts/generic ===
```

Comment:
This is still valid. There's just a typo in the directory name. should read gdal/ogr/ogrsf_frmts/generic

Reply:
Thanks. I found all of the missing directories with an `awk` script, so didn't check manually.